curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md

```diff
@@ -9,7 +9,7 @@ forumTopicId: 301417
 ## Description
 <section id='description'>
 Conditional rendering is useful, but you may need your components to render an unknown number of elements. Often in reactive programming, a programmer has no way to know what the state of an application is until runtime, because so much depends on a user's interaction with that program. Programmers need to write their code to correctly handle that unknown state ahead of time. Using <code>Array.map()</code> in React illustrates this concept.
-For example, you create a simple "To Do List" app. As the programmer, you have no way of knowing how many items a user might have on their list. You need to set up your component to <em><strong>dynamically render</strong></em> the correct number of list elements long before someone using the program decides that today is laundry day.
+For example, you create a simple "To Do List" app. As the programmer, you have no way of knowing how many items a user might have on their list. You need to set up your component to <dfn>dynamically render</dfn> the correct number of list elements long before someone using the program decides that today is laundry day.
```

Comment:
Remove the dfn tags here.

Reply:
Should I revert to using `<em><strong>`, for emphasis?

Reply:
No, I would just remove the `dfn` tags and not add any additional tags.